src/input-nested-select/input-nested-select.tsx

```diff
@@ -68,8 +68,8 @@ export const InputNestedSelect = <V1, V2, V3>({
     // CONST, STATE
     // =========================================================================
     const options = _options as NestedDropdownListItemProps<V1 | V2 | V3>[];
-    const [selectedKeyPaths, setSelectedKeyPaths] = useState<string[][]>(
-        _selectedKeyPath ? [_selectedKeyPath] : []
+    const [selectedKeyPaths, setSelectedKeyPaths] = useState<Set<string>>(
+        _selectedKeyPath ? new Set<string>([_selectedKeyPath.join(",")]) : new Set<string>()
```

Comment:
also suggest to have a common helper function to convert the key path array to Set